packages/perseus-editor/src/editor-page.tsx

```diff
@@ -83,15 +86,20 @@ class EditorPage extends React.Component<Props, State> {
     constructor(props: Props) {
         super(props);
 
+        // Convert any widgets that need to be converted to newer widget types
+        let convertedQuestionJson: PerseusRenderer = props.question;
+        if (props.question) {
+            convertedQuestionJson = convertDeprecatedWidgets(props.question);
+        }
+
+        const json = {
+            ..._.pick(this.props, "answerArea", "hints", "itemDataVersion"),
+            question: convertedQuestionJson,
+        };
```

Comment:
In an effort to remove the underscore library, could this be changed to:
```
const json = {
    answerArea: this.props.answerArea,
    hints: this.props.hints,
    itemDataVersion: this.props.itemDataVersion,
    question: convertedQuestionJson,
}
```

Reply:
Yeah good call, let's do it!